includes/rest/class-followers.php

```diff
@@ -78,7 +78,7 @@ public static function get( $request ) {
 		$json->actor = \get_author_posts_url( $user_id );
 		$json->type = 'OrderedCollectionPage';
 
-		$json->partOf = \get_rest_url( null, sprintf( '/%s/users/%d/followers', ACTIVITYPUB_REST_NAMESPACE, $user_id ) ); // phpcs:ignore
+		$json->partOf = get_rest_url_by_path( sprintf( 'users/%d/followers', $user_id ) ); // phpcs:ignore
```

Comment:
get_rest_url_by_path is in the Activitypub namespace, so you have to add use function Activitypub\get_rest_url_by_path as the top of the class.